package.json

```diff
@@ -20,31 +20,28 @@
   },
   "homepage": "https://github.com/KyberNetwork/smart-contracts#readme",
   "dependencies": {
-    "@nomiclabs/buidler-truffle5": "^1.0.1",
-    "@nomiclabs/buidler-web3": "^1.0.1",
-    "bignumber.js": "^5.0.0",
-    "braces": "^2.3.2",
-    "chai-as-promised": "^7.1.1",
-    "chai-bignumber": "^3.0.0",
-    "compare-versions": "^3.1.0",
-    "ganache-cli": "^6.8.0-istanbul.0",
-    "mathjs": "^4.3.0",
-    "rlp": "^2.0.0",
-    "secp256k1": "^3.7.1",
-    "sha3": "^2.0.6",
+    "@nomiclabs/buidler-truffle5": "1.0.2",
+    "chai-as-promised": "7.1.1",
+    "chai-bignumber": "3.0.0",
```

Comment:
Do we still need this? Since we are using chai-bn for tests in the web3 version upgrade

Reply:
u r right.
removed.